### ISSUE TITLE
Less OP version of Rearm

### ISSUE THREAD

**Issue by TheRisen41** [ash47/LegendsOfDota#314]
What do you think of adding a weaker version of rearm with a cooldown to the normal(default) mode of LoD like you did with multicast. So it works simillar to refresher orb.

**Comment by darklordabc** [comment]
I like this idea a lot.

**Comment by ash47** [comment]
Your name isn't @Jexah :O

**Comment by ash47** [comment]
Another alternate might be a version that only refreshes normal abilities
and non-global abilities -- no ults or globals (e.g. sun strike)

**Comment by darklordabc** [comment]
In the original LoD for wc3, they have a nerfed version, but its not nerfed enough in my opinion. It has a cooldown of 9/6/3. My friend has made a pull request that implements a working nerfed verison, with cooldowns of 60,30,15. I think this is a pretty good compromise, but there may be a more perfect nerf.

**Comment by darklordabc** [comment]
In general, I think its a pretty good idea to introduced nerfed versions of abilities that are so powerful they never get used. Maybe also add a LoD warning, saying they are custom abilities, and saying what is changed about them.

**Comment by Myrl** [comment]
@ash47 Now that I think about it, that also sounds good.

**Comment by TheRisen41** [comment]
I like ash's idea too but wouldn't that just be like adding all ults/global abilities + rearm to the troll combo list?

**Comment by Myrl** [comment]
Wouldn't that disallow one to have more than 1 ult? Correct me if I'm
wrong, but doesn't six skills mode require 2 ults?

**Comment by TheRisen41** [comment]
Yeah you're right. Didn't thought about that.

**Comment by Myrl** [comment]
I made another [pull request](https://github.com/ash47/LegendsOfDota/pull/319) since I do not know how to use git and I committed to master.

**Comment by Jexah** [comment]
Yeah, the trouble with adding modified versions of skills is that it confuses the hell out of players. They pick a skill expecting something, get something else, rage about how shit the game mode is, then leave. The worst part is, that in this case, you can't blame them. 

If a modified version if Rearm was to be implemented, 60s cooldown and not refresh ults could work.

**Comment by ash47** [comment]
Perhaps giving it a different logo as well so it's really different?

**Comment by darklordabc** [comment]
//Yeah, the trouble with adding modified versions of skills is that it confuses the hell out of players. //
This can be fixed by ability warnings. 
//If a modified version if Rearm was to be implemented, 60s cooldown and not refresh ults could work.//
Would'nt this be too much of a nerf? Its already got a long cooldown, only refreshing non-ultimate abilities would make it kind of pointless. There is not many builds that could benefit from having just lvl 1 abilities refreshed every 60 seconds. 

If we do go down the no ultimate routes, I think it should be specific ultimates that dont get refreshed, like mass wards, not all ultimates by default.

**Comment by Jexah** [comment]
You'd think it would be as easy as adding ability warnings, and it is for additional effects that are not advertised in Dota, because the skill isn't nerfed, because 99% of people don't read anything. 

That is not a solution to the issue.

We actually trialled this version of Rearm for a few months, where every global skill and some ults were banned combos. It was still really OP.

**Comment by darklordabc** [comment]
//That is not a solution to the issue.//
As well as an ability warning in the tooltip, there could be some graphic to the icon to make it obvious it is different. 
With the extra 2 essence shift @Myrl is working on, they have different tints to subtly show they are different skills.
![image](https://cloud.githubusercontent.com/assets/16277198/15763656/acbae336-2968-11e6-9397-16417fa8636c.png)

Even this would do, although its kind of ugly
![image](https://cloud.githubusercontent.com/assets/16277198/15763761/605e74fc-2969-11e6-95e6-1d12bef1d4a5.png)

.

**Comment by darklordabc** [comment]
//We actually trialled this version of Rearm for a few months, where every global skill and some ults were banned combos. It was still really OP.//
What was OP? People doing two tombstones? Two Zeus ultimates?

**Comment by darklordabc** [comment]
Okay I think I've figured out a pretty good compromise. 

Using rearm only lowers the cooldown of abilities by 30 seconds. For low cooldown abilities, the notice will be unnoticeable, but with long cooldown abilities, like Phantasm, it will only knock 30 seconds off. This has the benefit of making rearm still useful with other ults, but nowhere near as powerful as it was before. 

[Here is the code](http://hastebin.com/) for how it would work, and I've tested it, and it works.

**Comment by TheRisen41** [comment]
![tinker rearm2](https://cloud.githubusercontent.com/assets/8276667/15782529/03499b60-29ab-11e6-8227-f02b240f3b38.png)
Made a rearm icon :). Use it if you want

**Comment by Myrl** [comment]
@TheRisen41 Looks nice but doesn't look too dota2ish.

**Comment by TheRisen41** [comment]
@Myrl hehe guess you're right about that ^^

**Comment by TheRisen41** [comment]
@darklordabc this seems like a very good solution. Only ability that comes to my mind that could be abused is sunstrike and maybe rocketflare.

**Comment by TheRisen41** [comment]
![rearm_icon_green](https://cloud.githubusercontent.com/assets/8276667/15801925/4b0d0618-2aa4-11e6-995b-c7cefd9ebca9.png)
Or maybe just some recoloring?

**Comment by darklordabc** [comment]
I like that a lot.

**Comment by Myrl** [comment]
Yeah, I like it too.
